src/repl.ts

```diff
@@ -230,6 +231,19 @@ function enterPasteMode() {
   })
 }
 
+function loadFile(filename: string) {
+  try {
+    console.log(`// loading file ${filename}`.cyan)
+    let file_contents: string = fs.readFileSync(filename, 'utf8')
+    console.log(colorize(file_contents))
```

Comment:
this repl has been quite different from nodejs' repl. I don't know keeping same with node is good.